connector/openshift/openshift.go

```diff
@@ -61,22 +70,23 @@ type user struct {
 func (c *Config) Open(id string, logger log.Logger) (conn connector.Connector, err error) {
 	ctx, cancel := context.WithCancel(context.Background())
 
-	wellKnownURL := strings.TrimSuffix(c.Issuer, "/") + "/.well-known/oauth-authorization-server"
+	wellKnownURL := strings.TrimSuffix(c.Issuer, "/") + wellKnownURLPath
 	req, err := http.NewRequest(http.MethodGet, wellKnownURL, nil)
 
 	openshiftConnector := openshiftConnector{
-		apiURL:       c.Issuer,
-		cancel:       cancel,
-		clientID:     c.ClientID,
-		clientSecret: c.ClientSecret,
-		insecureCA:   c.InsecureCA,
-		logger:       logger,
-		redirectURI:  c.RedirectURI,
-		rootCA:       c.RootCA,
-		groups:       c.Groups,
-	}
-
-	if openshiftConnector.httpClient, err = newHTTPClient(c.InsecureCA, c.RootCA); err != nil {
+		apiURL:          c.Issuer,
+		cancel:          cancel,
+		clientID:        c.ClientID,
+		clientSecret:    c.ClientSecret,
+		insecureCA:      c.InsecureCA,
+		logger:          logger,
+		redirectURI:     c.RedirectURI,
+		rootCA:          c.RootCA,
```

Comment:
Since `openshiftConnector` is an internal type with no serializability or compatibility requirements, why not unify `rootCA` and `trustedCertPool` into a single `trustedCertPool` field? However, please see the overall comment regarding the `trustedCertPool` change.